src/components/ActivityFeed/ActivityFeed.js

```diff
@@ -228,7 +228,13 @@ class ActivityFeed extends React.Component<Props, State> {
         decimals = 18,
         iconUrl,
       } = assets.find(({ symbol }) => symbol === notification.asset) || {};
-      const value = utils.formatUnits(new BigNumber(notification.value.toString()).toFixed(), decimals);
+      let bigNumber;
+      try {
+        bigNumber = new BigNumber(notification.value.toString());
```

Comment:
`bigNumber = new BigNumber(get(notification, 'value', 0).toString());`
and no need for try..catch

Reply:
That would handle the `undefined` scenario, right? What do you think about the decimals? Is that something we need to take care of?

Reply:
I wouldn't care about decimals here